<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plugins_dialog</class>
 <widget class="QDialog" name="plugins_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="plugin_table">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>271</width>
     <height>266</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>271</width>
     <height>266</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>271</width>
     <height>266</height>
    </size>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>  1  </string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>  2  </string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>  3  </string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>  4  </string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>  5  </string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>  6  </string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>  7  </string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Plug-in</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Loaded</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>                   Plug-in</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>  Loaded</string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="4" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="5" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="6" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item row="7" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="load1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>53</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 8px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="load2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>83</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 8px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="load3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>113</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 8px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="load4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>143</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 8px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="load5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>173</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 8px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="load6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>203</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 8px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="load7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>233</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
 border: 2px solid gray;
 border-radius: 8px;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>296</y>
     <width>256</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="open">
      <property name="minimumSize">
       <size>
        <width>76</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>76</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Open folder </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Load plugin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="remove">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Remove plugin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>load</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>plugin_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
